README.md

```diff
@@ -1,21 +1,47 @@
 # GameCI
 
 Documentation for open source GameCI projects. You can find the `.md` files inside the [docs](./docs) folder.
-The live version is available on [game.ci](https://game.ci).
+The live version is available on [game.ci](https://game.ci). This website is built using 
+[Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-## Run the documentation locally
+### Installation
 
-You will need `NodeJS v12.x` and the latest
-[yarn](https://classic.yarnpkg.com/en/docs/install/).
+```bash
+yarn
+```
 
-1. Clone repository
-2. Run the following commands in the folder to install requirements and run the development server
+### Local Development
 
 ```bash
-yarn
-yarn dev
+yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+### Build
+
+```bash
+yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+Using SSH:
+
+```bash
+USE_SSH=true yarn deploy
 ```
 
+Not using SSH:
+
+```bash
+GIT_USER=<Your GitHub username> yarn deploy
+```
```

Comment:
These are not cross-platform compatible and we wouldn't deploy from local anyway. If anything this should probably move to `DEVELOPMENT.md`, as the readme is more for the public to see.

Reply:
This is docusaurus' default readme content, agreed about deployment, we're using vercel anyway for deployment

Reply:
I removed the `Deployment` section from readme (since we didn't have it before anyway). Only kept what's relevant